src/ops/terraform/terraform_cmd_generator.py

```diff
@@ -27,6 +31,11 @@ def __init__(self, root_dir, cluster_config, inventory_generator, ops_config, te
         self.template = template
 
     def generate(self, args):
+        if 'AWS_PROFILE' in os.environ:
```

Comment:
Is this env var set by opscli before running tf?

Reply:
This is one of the default variables used by the aws cli. https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
Also, for terraform, I don't think we're using `AWS_PROFILE`. terraform fetches this data from the `provider.tf` file, which is generated via `himl`

Reply:
What should I use to differentiate between AWS and other cloud providers?

Reply:
@serban21 can you chime in pls?